manuscript/1-prehistory.md

```diff
@@ -1,6 +1,6 @@
 {section: mainmatter}
 # Prehistory
-My start in community started with AdminScripting.com, meant as an adjunct to my first bestselling book, _Managing Windows with VBScript and WMI_. It started simply, as more or less a blog, running on PHP-Nuke. But shortly after, I had a conversation with Jeremy Moskowitz, who'd just launched GPOAnswers.com to support his won work (it's now GPAnswers.com). Our colleague, Brett Hill, had launches IISAnswers.com, and the two of them had a shared landing page on GetTheAnswers.com (which Jeremy still owns). I loved the naming, and ScriptingAnswers.com was born.
+My start in community started with AdminScripting.com, meant as an adjunct to my first bestselling book, _Managing Windows with VBScript and WMI_. It started simply, as more or less a blog, running on PHP-Nuke. But shortly after, I had a conversation with Jeremy Moskowitz, who'd just launched GPOAnswers.com to support his own work (it's now GPAnswers.com). Our colleague, Brett Hill, had launched IISAnswers.com, and the two of them had a shared landing page on GetTheAnswers.com (which Jeremy still owns). I loved the naming, and ScriptingAnswers.com was born.
```

Comment:
As github isn't doing a great job of showing the changes, they are just:

- changed "to support his **won** work" to "to support his **own** work"
- changed "had **launches** IISAnswers.com" to "had **launched** IISAnswers.com"